### ISSUE TITLE
Create A11y tests for Carbon React components

### ISSUE THREAD

**Issue by dakahn** [carbon-design-system/carbon#4611]
As the A11y Project all-clears a component we also want to introduce rudimentary accessibility testing to protect against regressions. We're doing this on a component by component basis using the end to end helpers introduced in PR #4575

**Comment by joshblack** [comment]
Hey @dakahn! :wave: Going to close this one out, let me know if you'd like me to keep it open!
